### DISCUSSION TITLE
R5.2 UCI open issues in sunbird ED env

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [samagra-comms/community#27] [General Discussion]
@ ChakshuGautam  @aayushkedia95 There are two issues open in 5.2 sunbird ED regression, can u please look into those two Issues and provide ETA:
https://project-sunbird.atlassian.net/browse/ED-1518
https://project-sunbird.atlassian.net/browse/ED-1484

@Krishnaj20 @chitranshu-keshav @vinukumar-vs

**Comment by ashokreddy1208** [comment]
@ChakshuGautam @aayushkedia95 Can you please respond to this issues.

Note:- we need to provide signoff for R5.2 sunbird ED today

cc : @Krishnaj20 @chitranshu-keshav @vinukumar-vs

**Comment by Naveenkumar739** [comment]
https://project-sunbird.atlassian.net/browse/ED-1518 - Initially I have created conversation listing with global tara odk form then updated with list button odk form but still i m getting response for global tara in watsupchatbot

https://project-sunbird.atlassian.net/browse/ED-1484- If enter 99 then it is taking to main menu instead of previous menu

**Reply by abhay-samagra** [reply]
@ashokreddy1208, @Naveenkumar739 
There are problems in the Dev and staging environments. The contributors can't access these environments anymore.
As a result, the team has been working on Pre-prod environment. Issues have been fixed there and Kameswarrao is also aware.

**Comment by ashokreddy1208** [comment]
@abhay-samagra @ChakshuGautam please let me know your availability, we can connect with devops team to check logs and environment access related issues.
cc @ChakshuGautam @gandham-santhosh
